filterpage.html

```diff
@@ -0,0 +1,396 @@
+      let upwardArrow = document.querySelector(".arrow");
+      let makeListWrapper = document.querySelector(".make-list-wrapper");
+      upwardArrow.addEventListener("click", function (e) {
+        makeListWrapper.classList.toggle("collapse");
+        e.target.classList.toggle("downward");
+      });
+
+      //here we are checking the selected type is checkbox and we have checked the checkbox then we push it into array
+      //if we unselected the checkbox using splice the selected index is removed
+      makeListWrapper.addEventListener("change", function (event) {
+        if (event.target.matches('input[type="checkbox"]')) {
+          const checkbox = event.target;
+          if (checkbox.checked) {
+            selectedMakeFilter.push(checkbox.value);
+          } else {
+            const index = selectedMakeFilter.indexOf(checkbox.value);
+            if (index > -1) {
+              selectedMakeFilter.splice(index, 1);
+            }
+          }
+        }
+      });
+
+      //onclick of the apply filter the selection page hides and first page open and in result on first page we can see the selected status
+      let applyFilter = document.querySelector(".apply-filters__btn");
+      let result = document.querySelector(".selection-status");
+      applyFilter.addEventListener("click", function () {
+        selection.classList.add("hide");
+        openFirstPageWrapper.classList.remove("hide");
+        result.innerHTML =
+          selectedMakeFilter.length > 0
+            ? `<div>Make: <span class="selected-result">${selectedMakeFilter.join(
+                ", "
+              )} </span></div>`
+            : "Make: No selected filters";
+      });
+
+      //onclick of the arrow the Body list open and collapse
+      let bodyTypeArrow = document.querySelector(".body-arrow");
+      let bodyListWrapper = document.querySelector(".body-list-wrapper");
+      bodyTypeArrow.addEventListener("click", function (e) {
+        bodyListWrapper.classList.toggle("collapse");
+        e.target.classList.toggle("downward");
+      });
+
+      //here we are checking the selected type is checkbox and we have checked the checkbox then we push it into array
+      //if we unselected using splice the selected index is removed
+      bodyListWrapper.addEventListener("change", function (event) {
+        if (event.target.matches('input[type="checkbox"]')) {
+          const checkbox = event.target;
+          if (checkbox.checked) {
+            selectedBodyFilter.push(checkbox.value);
+          } else {
+            const index = selectedBodyFilter.indexOf(checkbox.value);
+            if (index > -1) {
+              selectedBodyFilter.splice(index, 1);
+            }
+          }
+        }
+      });
+
+      //onclick of the apply filter the selection page hides and first page open and in BodyTypeResult on first page we can see the selected status
+      let BodyTypeResult = document.querySelector(".body-selection-status");
```

Comment:
Use camelCase convention.
```suggestion
      let bodyTypeResult = document.querySelector(".body-selection-status");
```